charts/infra-apps/Chart.yaml

```diff
@@ -17,12 +17,18 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "argo-cd: update from 2.6.7 to 2.7.0"
+      description: "argo-cd: update from 2.7.0 to 2.7.1"
       links:
         - name: GitHub Release
-          url: https://github.com/argoproj/argo-cd/releases/tag/v2.7.0
-    - kind: changed
-      description: "argo-cd: update chart from 5.29.1 to 5.30.0"
+          url: https://github.com/argoproj/argo-cd/releases/tag/v2.7.1
+    - kind: security
+      description: "argo-cd: update chart from 5.30.0 to 5.32.0"
       links:
-        - name: "feat: Upgrade Argo CD to v2.7.0"
-          url: https://github.com/argoproj/argo-helm/pull/1989
+        - name: "feat: allow extraObjects to contain string template"
+          url: https://github.com/argoproj/argo-helm/pull/1993
+        - name: "chore: Update Argo CD to v2.7.1"
+          url: https://github.com/argoproj/argo-helm/pull/1993
+        - name: "feat: Use tini as the ENTRYPOINT"
+          url: https://github.com/argoproj/argo-helm/pull/1993
```

Comment:
All the same PR?

Reply:
ups no, i failed at copypasting :(

Reply:
I saw it when it was in draft but did not say anything as I thought you were still working on it